osgp/platform/osgp-adapter-ws-shared/src/main/java/org/opensmartgridplatform/adapter/ws/shared/services/AbstractResendNotificationService.java

```diff
@@ -174,7 +186,22 @@ private long calculateDelay(final int earlierResentNotifications) {
         return delay;
     }
 
-    public abstract void resendNotification(ResponseData responseData);
+    // public void resendNotification(ResponseData responseData);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)